.env.testing

```
@@ -20,7 +20,12 @@ LOG_STACK=single
 LOG_DEPRECATIONS_CHANNEL=null
 LOG_LEVEL=debug
 
-DB_CONNECTION=sqlite
+DB_CONNECTION=mysql
+DB_HOST=db
+DB_PORT=3306
+DB_DATABASE=db
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_DATABASE key should go before the DB_HOST key

(UnorderedKey)